### PR TITLE
[14.0][MIG] stock request picking type

### PR DESCRIPTION
**Pull request by ReedHayashikawa**

### CODE REVIEW

File: stock_request_picking_type/__manifest__.py [hunk 1]
```diff
@@ -4,9 +4,9 @@
 {
     "name": "Stock Request Picking Type",
     "summary": "Add Stock Requests to the Inventory App",
-    "version": "13.0.1.0.0",
+    "version": "14.0.1.0.0",
```
Comment (rousseldenis) [inline stock_request_picking_type/__manifest__.py:7]: I think you didn't run the migration flow as described. This should be in [MIG] commit.

Don't bother this time.

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by ReedHayashikawa** [comment]
@patrickrwilson @osi-scampbell
Could you assist with the review

**Comment by patrickrwilson** [comment]
@ReedHayashikawa can you look at the pre-commit and fix that? Then I'll review.

**Comment by rousseldenis** [comment]
@ReedHayashikawa Could you check Travis ?

**Review by patrickrwilson — APPROVED** [review]
Tested, works as intended. LGTM

**Pull request merged**